.deny.toml

```diff
@@ -23,6 +23,7 @@ allow = [
   "CC0-1.0",
   "Unlicense",
   "Zlib",
+  "Unicode-3.0",
```

Comment:
@dae, I added this line to solve the problem. Is it acceptable?

Reply:
Oops, I get a new error:


error[custom]: unknown term
--
  | ┌─ /state/build/lin-ci/ankitects/anki-ci/.deny.toml:26:4
  | │
  | 26 │   "Unicode-3.0",
  | │    ^^^^^^^^^^^
  |  
  | 2024-07-15 12:39:03 [ERROR] failed to deserialize config from '/state/build/lin-ci/ankitects/anki-ci/.deny.toml'

Reply:
try

```
[Unicode License v3](https://spdx.org/licenses/Unicode-3.0.html)
```

ref https://github.com/EmbarkStudios/cargo-deny/blob/main/deny.template.toml#L89C7-L89C33

and https://spdx.org/licenses/

Reply:
Confusing, <https://github.com/EmbarkStudios/cargo-deny/blob/6344cc566621410a0865632b4ef0e82a20408676/deny.template.toml#L89C7-L89C33> said it's spdx license, but `"Unicode-3.0"` is spdx format.

Reply:
oh, this may be the problem. <https://github.com/EmbarkStudios/spdx/pull/65/files#diff-94815ebc8dd748656c1a4414adabdbbc8eeefcf0e1b1fc069223bbdf8bf08974R1843 it's new added.>

Reply:
well, 
the 3.0 available from spdx 0.10.4
cargo deny uses "0.10", so I guess here is the problem.

<https://github.com/EmbarkStudios/cargo-deny/blob/6344cc566621410a0865632b4ef0e82a20408676/Cargo.toml#L103>

Reply:
I guess install the latest cargo-deny could fix this. Which is now `0.14.24`.

Reply:
lol

Reply:
Yes, all good now - thank you both.